src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -0,0 +1,416 @@
+        } else {
+            $result = $s3->GetObject([
+                'Bucket' => $bucket,
+                'Key' => $dbFilename,
+            ]);
+            $fp = fopen($dbFilename, 'wb');
+            stream_copy_to_stream($result->getBody()->getContentAsResource(), $fp);
+            $this->say("Database dump file downloaded >>> $dbFilename");
+        }
+        return $dbFilename;
+    }
+
+    /**
+     * Configure AWS credentials.
+     *
+     * @param string $awsConfigDirPath
+     *   Path to the AWS configuration directory.
+     * @param string $awsConfigFilePath
+     *   Path to the AWS configuration file.
+     */
+    protected function configureAwsCredentials(string $awsConfigDirPath, string $awsConfigFilePath)
+    {
+        $yes = $this->io()->confirm('AWS S3 credentials not detected. Do you wish to configure them?');
+        if (!$yes) {
+            return Result::cancelled();
+        }
+
+        if (!is_dir($awsConfigDirPath)) {
+            $this->_mkdir($awsConfigDirPath);
+        }
+
+        if (!file_exists($awsConfigFilePath)) {
+            $this->_touch($awsConfigFilePath);
+        }
+
+        $awsKeyId = $this->ask("AWS Access Key ID:");
+        $awsSecretKey = $this->askHidden("AWS Secret Access Key:");
+        return $this->taskWriteToFile($awsConfigFilePath)
+            ->line('[default]')
+            ->line("aws_access_key_id = $awsKeyId")
+            ->line("aws_secret_access_key = $awsSecretKey")
+            ->run();
+    }
+
+    /**
+     * Refresh a site database in Lando.
+     *
+     * @param string $siteName
+     *   The Drupal site name.
+     *
+     * @return \Robo\Result
+     *   The result of the set of tasks.
+     */
+    public function databaseRefreshLando(string $siteName = 'default'): Result
+    {
+        $this->io()->title('lando database refresh.');
+
+        $dbPath = $this->databaseDownload($siteName);
+
+        $this->io()->section("importing $siteName database.");
+        $this->say("Importing $dbPath");
+        // If this is a multi-site, include a host option so Lando imports to the correct database.
+        $hostOption = $siteName !== 'default' ? "--host=$siteName" : '';
+        $this->taskExec('lando')
+            ->arg('db-import')
+            ->arg($dbPath)
+            ->arg($hostOption)
+            ->run();
+
+        $this->say("Deleting $dbPath");
+        $this->taskExec('rm')->args($dbPath)->run();
+        return $this->drushDeployLando($siteName);
+    }
+
+    /**
+     * Generate Drupal login link.
+     *
+     * @param string $siteDir
+     *   The Drupal site directory name.
+     * @param bool $lando
+     *   Use lando to call drush, else call drush directly.
+     *
+     * @aliases uli
+     *
+     * @return \Robo\Result
+     *   The result of the set of tasks.
+     */
+    public function drupalLoginLink($siteDir = 'default', $lando = true): Result
```

Comment:
```suggestion
    public function drupalLoginLink($siteDir = 'default', $lando = true): Result {
```
We want this, right?

Reply:
I see you are doing this elsewhere too. Is this intentional?

Reply:
See comment above.